src/server/Document/component.jsx

```diff
@@ -72,7 +77,7 @@ const Document = ({
           </>
         )}
       </head>
-      <body>
+      <body {...ampGeoPendingAttrs}>
```

Comment:
I think the only way to test that the pending class is removed by the AMP Geo component and the ads display once that is returned is to actually run this in a browser (using Cypress). We could add this as a test for a page with display advertisement.

Reply:
Sounds like a good idea